features/in_onderzoek.feature

```diff
@@ -12,109 +12,235 @@ Functionaliteit: in onderzoek
+      | inOnderzoek.voornamen     | true    |
+      | inOnderzoek.voorletters   | true    |
+      | inOnderzoek.volledigeNaam | true    |
+      En heeft de persoon een 'ouder' met de volgende gegevens
+      | naam                              | waarde |
+      | naam.voornamen                    | Noa    |
+      | naam.voorletters                  | N.     |
+      | geslacht.code                     | V      |
+      | geslacht.omschrijving             | vrouw  |
+      | ouderAanduiding                   | 1      |
+      En heeft de persoon een 'ouder' met de volgende gegevens
+      | naam                              | waarde |
+      | naam.voornamen                    | Jan    |
+  	  | naam.voorletters                  | J.     |
+      | geslacht.code                     | M      |
+      | geslacht.omschrijving             | man    |
+      | ouderAanduiding                   | 2      |
+      En heeft de persoon een 'partner' met de volgende gegevens
+      | naam                              | waarde          |
+  	  | burgerservicenummer               | 000000127       |
+      | naam.voornamen                    | Karel           |
+      | geslacht.code                     | M               |
+      | geslacht.omschrijving             | man             |
+      | aangaanHuwelijkPartnerschap.type  | Datum           |
+      | aangaanHuwelijkPartnerschap.datum | 2009-11-02      |
+      | aangaanHuwelijkPartnerschap.datum | 2 november 2009 |
 
-    @gba
     Scenario: persoon heeft gezagInOnderzoek en persoonInOnderzoek
-      Gegeven de persoon met burgerservicenummer 555550001 heeft de volgende persoonsgegevens in de registratie
-      | categorie | burgerservicenummer (01.20) | voornamen (02.10) | voorvoegsel (02.30) | geboortedatum (03.10) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) | datum einde onderzoek (83.30) |
-      | 1         | 555550001                   | Arnitta           |                     | 19231213              | 010410                          | 20120920                       |                               |
-      En de persoon met burgerservicenummer 555550001 heeft de volgende gezagsverhoudinggegevens in de registratie
-      | Categorie | indicatie curatele (33.10) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) | datum einde onderzoek (83.30) |
-      | 11        | 1                          | 113310                          | 20101205                       |                               |
+      Gegeven een persoon heeft de volgende 'persoon' gegevens
+      | burgerservicenummer (01.20) | voornamen (02.10) | geboortedatum (03.10) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) |
+      | 000000139                   | Arnitta           | 19231213              | 010410                             | 20120920                    |
+      En de persoon heeft nog een 'gezagsverhouding' met de volgende gegevens
```

Comment:
is dit de manier om dit toe te voegen? "En de persoon heeft nog een 'gezagsverhouding' met de volgende gegevens"

Ik had hier verwacht dat "En de persoon heeft de volgende 'gezagsverhouding' gegevens" zou worden gebruikt

Reply:
Jouw versie werkt helaas niet. 
Maar in principe heb je gelijk omdat de versie met 'nog' eigenlijk bedoelt is voor de situatie waarbij er meerdere occurences van dezelfde type groep bestaan, dus bijvoorbeeld meerdere kinderen of ouders.